reStream.sh

```diff
@@ -143,12 +150,24 @@ case "$rm_version" in
             fb_file="/dev/shm/swtfb.01"
             pixel_format="rgb565le"
         else
+            # Get the Remarkable software version before proceeding (File date of xochitl.)
             width=1872
             height=1404
-            bytes_per_pixel=1
             fb_file=":mem:"
-            pixel_format="gray8"
-            video_filters="$video_filters,transpose=2"
+
+            # Use updated video settings?
+            if [ $(ssh_cmd 'stat -c "%Y" /usr/bin/xochitl') -ge $rm2_old_xochitl_date ]; then
```

Comment:
This is rather brittle if the user does something to modify the xochitl binary. It would probably be a better idea to parse the version number with `grep REMARKABLE_RELEASE_VERSION= /usr/share/remarkable/update.conf`

Reply:
Yes, I agree, thank you. I've fixed this in my latest commit.